File: Constructorio_Net/src/models/FilterSortOption.java
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

/**
 * Constructor.io Sort Option
 **/
namespace Constructorio_NET
{
    public class FilterSortOption {
  
        [JsonPropertyName("display_name")]
        public String displayName;

        [JsonPropertyName("sort_by")]
        public String sortBy;

        [JsonPropertyName("sort_order")]
        public String sortOrder;
  
        [JsonPropertyName("status")]
        public String status;

    }  
}

File: src/Constructorio_NET/models/common/FilterSortOption.cs
using System;
using Newtonsoft.Json;

/**
 * Constructor.io Sort Option
 */
namespace Constructorio_NET.Models
{
    public class FilterSortOption
    {
        [JsonProperty("display_name")]
        public string DisplayName { get; set; }

        [JsonProperty("sort_by")]
        public string SortBy { get; set; }

        [JsonProperty("sort_order")]
        public string SortOrder { get; set; }

        [JsonProperty("status")]
        public string Status { get; set; }
    }
}